Todo-Redux/src/js/modules/prelogin/route.js

```diff
@@ -0,0 +1,21 @@
+import '../../../static/stylesheets/style';
```

Comment:
You can add path aliases in Webpack file, and can directly import any file by its alias.

`import 'stylesheets/style';`

It looks cleaner and doesn't need to remember its location from the current file.